### PR TITLE
Remove alibaba support

### PR DESCRIPTION
**Pull request by janosSarusiKis**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| API breaks?     | yes
| Deprecations?   | yes
| Related tickets |
| License         | Apache 2.0

### What's in this PR?
Complete remove of Alibaba support.

- Alibaba, ACK, OSS related packages were removed.
- Depended code were removed or updated.
- Migrations were added

### Why?
We no longer support Alibaba.

### Checklist

- [x] Implementation tested (with at least one cloud provider)
- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] OpenAPI and Postman files updated (if needed)
- [x] User guide and development docs updated (if needed)

### CODE REVIEW

**Comment by nmarcetic** [comment]
@janosSarusiKis I didn't catch any public discussion about removing Alibaba support, is there such tread or any info that explains/announce removal? Thanks.

**Comment by janosSarusiKis** [comment]
> @janosSarusiKis I didn't catch any public discussion about removing Alibaba support, is there such tread or any info that explains/announce removal? Thanks.

Hi! The decision was made internally after discussing it with key customers.

**Comment by nmarcetic** [comment]
> > @janosSarusiKis I didn't catch any public discussion about removing Alibaba support, is there such tread or any info that explains/announce removal? Thanks.
> 
> Hi @nmarcetic ! The decision was made internally after discussing it with key customers.

Go ti. Thanks for clarification @janosSarusiKis

**Review by pregnor — APPROVED** [review]

File: src/cluster/manager_common_updater.go [hunk 1]
```diff
@@ -282,7 +282,6 @@ func (c *commonUpdater) Update(ctx context.Context) error {
 }
 
 // labelNodesWithNodePoolName add node pool name labels for all nodes.
-// It's used only used in case of ACK etc. when we're not able to add labels via API.
 func labelNodesWithNodePoolName(ctx context.Context, commonCluster CommonCluster) error {
```
Comment (orymate) [inline src/cluster/manager_common_updater.go:285]: do we use this anywhere?

**Review by orymate — APPROVED** [review]

File: src/cluster/manager_common_updater.go (hunk 1 above)
Comment (janosSarusiKis) [reply src/cluster/manager_common_updater.go:285]: Just above in line 278 inside the Update func.

File: database/migrations/mysql/1611581486_drop_alibaba_tables.down.sql [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  `master_system_disk_size` int(11) DEFAULT NULL,
+  `snat_entry` tinyint(1) DEFAULT NULL,
+  `ssh_flags` tinyint(1) DEFAULT NULL,
+  `kubernetes_version` varchar(255),
+  `v_switch_id` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+UPDATE `alibaba_acsk_clusters` SET `v_switch_id` = NULL;
+
```
Comment (sagikazarmark) [inline database/migrations/mysql/1611581486_drop_alibaba_tables.down.sql:49]:
```suggestion
```

File: database/migrations/mysql/1611581486_drop_alibaba_tables.up.sql [hunk 3]
```diff
@@ -0,0 +1,3 @@
+DROP TABLE IF EXISTS `alibaba_acsk_clusters`;
+DROP TABLE IF EXISTS `alibaba_acsk_node_pools`;
+DROP TABLE IF EXISTS `alibaba_buckets`;
```
Comment (sagikazarmark) [inline database/migrations/mysql/1611581486_drop_alibaba_tables.up.sql:3]:
```suggestion
DROP TABLE IF EXISTS `alibaba_buckets`;

```

File: database/migrations/postgres/1611582374_drop_alibaba_tables.down.sql [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  "zone_id" text,
+  "master_instance_type" text,
+  "master_system_disk_category" text,
+  "master_system_disk_size" integer,
+  "snat_entry" boolean,
+  "ssh_flags" boolean,
+  "kubernetes_version" text,
+  "v_switch_id" text,
+  PRIMARY KEY ("id")
+);
```
Comment (sagikazarmark) [inline database/migrations/postgres/1611582374_drop_alibaba_tables.down.sql:49]:
```suggestion
);

```

File: database/migrations/postgres/1611582374_drop_alibaba_tables.up.sql [hunk 5]
```diff
@@ -0,0 +1,3 @@
+DROP TABLE IF EXISTS "alibaba_acsk_clusters";
+DROP TABLE IF EXISTS "alibaba_acsk_node_pools";
+DROP TABLE IF EXISTS "alibaba_buckets";
```
Comment (sagikazarmark) [inline database/migrations/postgres/1611582374_drop_alibaba_tables.up.sql:3]:
```suggestion
DROP TABLE IF EXISTS "alibaba_buckets";

```

**Review by sagikazarmark — CHANGES_REQUESTED** [review]

**Review by sagikazarmark — APPROVED** [review]

**Pull request merged**